docs/operate/reference/viam-server/manage-viam-server.md

```diff
@@ -156,6 +156,40 @@ brew services restart viam-server
 
 ## Update `viam-server`
 
+The steps to update `viam-server` differ depending on your installation method.
+You can check this by seeing if `viam-agent` is running.
+Run the following command:
+
+```sh {class="command-line" data-prompt="$" data-output="2"}
+ps aux | grep viam-agent
+root      566431  0.5  0.2 1247148 20336 ?       Ssl  11:24   0:00 /opt/viam/bin/viam-agent --config /etc/viam.json
+```
+
+If you see a process running viam-agent, then you used `viam-agent` to install `viam-server`.
+If not follow the steps for the standalone version.
+
+{{< tabs >}}
+{{% tab name="Installed with viam-agent" %}}
+
+By default, `viam-agent` automatically upgrades to the latest stable version of `viam-server`.
+To change this behaviour or change to a specific version, [change the `version_control` settings in your machine's settings](/manage/reference/viam-agent/#version_control-version-management-for-viam-agent-and-viam-server) in the [Viam app](https://app.viam.com/).
+For example:
+
+```json {class="line-numbers linkable-line-numbers" data-line=""}
+{
+  "agent": {
+    "version_control": {
+      "agent": "stable",
+      "viam-server": "dev"
+    }
+  },
+  "components": [ ... ]
+}
+```
+
```

Comment:
```suggestion

To use the new versions of `viam-server` and `viam-agent`, [restart `viam-agent`](/manage/reference/viam-agent/manage-viam-agent/).
When you stop or restart `viam-agent`, the agent also restarts `viam-server`.

```

Suggestion: visiting the `version_control` link informed me of this massive footgun, I think it's worth mentioning on this page even if it's redundant because otherwise a lot of users won't actually update either process until their next reboot!

Reply:
For some reason GitHub won't let me comment this on the single line beneath the code block, it's gotten into a weird state where it's stuck selecting multiple lines. So if you want to include this you might have to do it manually.